data/mods/gen9ssb/conditions.ts

```diff
@@ -352,6 +352,15 @@ export const Conditions: {[k: string]: ModdedConditionData & {innateName?: strin
 			this.add(`c:|${getName('hsy')}|https://www.youtube.com/watch?v=g104OJIh9hs`);
 		},
 	},
+	hydrostaticsuser: {
+		noCopy: true,
+		onFaint(pokemon) {
+			if (pokemon.side.pokemon.some(mon => mon.name === 'PartMan')) {
+				// Custom message for PartMan
+				this.add(`c:|${getName('PartMan')}|Hydro here have a tiara`);
+			}
+		},
+	},
```

Comment:
```suggestion
```
Looks like some work bled over, add this to the hydrostatics PR

Reply:
I'd kept it in this one since it was related to mine; not his - will update!

Reply:
Moved in https://github.com/smogon/ps-ssb/pull/86/commits/d7f7ffb22bdb6b19b4de14c174a6647f879b4652 and https://github.com/smogon/ps-ssb/pull/87/commits/7fc9b9752886ffa4ec3fa5431ff08d33ce39b812